src/hash_table.c

```diff
@@ -24,11 +24,11 @@ t_hash *hash_table_create(t_heap *heap, uint32_t size, hash_function *hf) {
 
 void hash_table_print(t_hash *ht) {
     printf("Start Table\n");
-    for (int i = 0; i < ht->size; i++) {
+    for (uint32_t i = 0; i < ht->size; i++) {
```

Comment:
also not sure about this here as i and ht->size were different types 🤔

Reply:
oh yeah, I hadn't refactored this function to work yet :/